src/pages/UsersPage.spec.js

```diff
@@ -0,0 +1,10 @@
+import { render, screen } from '@testing-library/react'
+import UsersPage from './UsersPage'
+
+describe('UserPage', () => {
+  it('renders', () => {
+    render(<UsersPage />)
+    const listItems = screen.getAllByRole('listitem')
+    expect(listItems).toHaveLength(57)
```

Comment:
I would probably check for the `User` component and how many should be rendered. In your case it's three cards.